Procfile

```diff
@@ -1,4 +1,4 @@
-web: gunicorn kuma.wsgi -b 0.0.0.0:8000
+web: gunicorn kuma.wsgi -w 2 -b 0.0.0.0:8000
```

Comment:
Interesting, are you saying Gunicorn was blocking on requests when it requested things from kumascript?